app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title = t("title")
```

Comment:
```suggestion
    base_title = t "title"
```
Nên bỏ () ở những chỗ có thể